### ISSUE TITLE
False Positive

### ISSUE THREAD

**Issue by Skeeve** [SIWECOS/InfoLeak-Scanner#17]
Bitte mal überprüfen.

M.W. werden Plugins nicht über <a> eingebunden ;)

Deswegen passt das Scanergebnis hier nicht so ganz.

> Das verwendete CMS Plugin wordfence in DOM-Node href mit dem Inhalt https://siwecos.de/tag/wordfence/ wurde erkannt. Dies stellt jedoch nicht direkt eine Schwachstelle dar und bedarf einer genaueren Prüfung. 

Siehe Screenshot:

![notanissue](https://user-images.githubusercontent.com/725404/44145752-288afe9a-a08d-11e8-8a45-7aba13e97b2a.png)

**Comment by y-ates** [comment]
Ohne es verifiziert zu haben, sollte es jetzt behoben sein :).

**Comment by Skeeve** [comment]
Ja, die tagwolke wird dynamisch generiert. Mein Bild war gefaket ;)
